chain/bitcoind_client.go

```diff
@@ -160,6 +160,13 @@ func (c *BitcoindClient) GetBlock(hash *chainhash.Hash) (*wire.MsgBlock, error)
 	return c.chainConn.GetBlock(hash)
 }
 
+// GetBlocksBatch returns batchBlocks from batchHashes.
```

Comment:
nit: use same Godoc here as in `BitcoindConn.GetBlocksBatch()` since that's a bit more verbose and clear?